adobe/spectrum-web-components
# Various Dev Time Perf Updates

## Description
- make CEM smaller, about 8.2MB down to 650KB!
- slim down the images in the Thumbnail stories, about 1.1MB down to 400KB
- turn the service worker in the Docs Site off at dev time.
- make CSS watching incremental and properly segment "build CSS" from "build on CSS file"
- enhance caching for low change files in dev time Storybook

## Motivation and context
Found these working through the work in #2257 

## How has this been tested?
- boot up the Docs Site locally
  - clear the service work from the Application cache
  - reload the site
- boot up Storybook
  - see that it doesn't redirect you to the Docs site in the service worker cache
  - see that it's much much faster

## Types of changes

## Checklist
